ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -954,7 +1066,7 @@ bb_decode (void **pdata, const GstTensorsConfig * config,
     return GST_FLOW_ERROR;
   }
 
-  /** reset the buffer with alpha 0 / black */
+                                                                /** reset the buffer with alpha 0 / black */
```

Comment:
Mistake?

Reply:
Indeed, not intentional.
This has been introduced on gst-indent commit, not sure why.

Fixed.